rustworkx/__init__.py

```diff
@@ -1490,7 +1490,7 @@ def cartesian_product(
 
 
 @_rustworkx_dispatch
-def bfs_search(graph, source, visitor):
+def bfs_search(graph, source=None, visitor=None):
```

Comment:
Because of the dispatch I think this has no impact but it looks like we had `source` as a required argument in the universal functions by accident?